src/executor/infer_graph_attr_pass.cc

```diff
@@ -416,11 +416,6 @@ nnvm::Graph InferStorageType(nnvm::Graph&& graph,
     DispatchModeVector dispatch_modes(graph.indexed_graph().num_nodes(), DispatchMode::kUndefined);
     graph.attrs["dispatch_mode"] = std::make_shared<any>(std::move(dispatch_modes));
   }
-  // initialize unknown values for dispatch modes
```

Comment:
why is this removed?

Reply:
i remember now. because i found there is duplicated code here. The code I delete is exactly the same as the code above.